app/src/test/scala/org/alephium/explorer/config/ExplorerConfigSpec.scala

```diff
@@ -36,8 +36,13 @@ class ExplorerConfigSpec extends AlephiumSpec with ScalaCheckDrivenPropertyCheck
 
   "ficus" should {
     "load config" in {
-      val typesafeConfig = ConfigFactory.load()
-      Try(typesafeConfig.as[ExplorerConfig]("alephium")) is a[Success[ExplorerConfig]]
+      // We make sure every config file is valid
+      Seq(NetworkId.AlephiumMainNet, NetworkId.AlephiumTestNet, NetworkId.AlephiumDevNet).foreach {
+        networkId =>
+          val typesafeConfig =
+            ConfigFactory.parseResources(s"application-${networkId.networkType}.conf").resolve()
+          Try(typesafeConfig.as[ExplorerConfig]("alephium")) is a[Success[ExplorerConfig]]
```

Comment:
[Nit] Shall we verify the value is correctly loaded for different network id?

Reply:
good idea, I'll add it to the test